src/configuration/types.rs

```diff
@@ -556,6 +556,8 @@ pub struct Configuration {
     pub import_declaration_space_surrounding_named_imports: bool,
     #[serde(rename = "jsxExpressionContainer.spaceSurroundingExpression")]
     pub jsx_expression_container_space_surrounding_expression: bool,
+    #[serde(rename = "jsxElement.spaceBeforeSelfClosingTag")]
```

Comment:
> For future changes, what's the expected flow like? I think something like this is pretty non-controversial, so maybe straight to PR, but for others should I raise an issue first to discuss?

If you go straight to a PR it's more at risk for something being rejected and higher risk I will feel bad rejecting it 😅

> Also, what's dprint's opinion in terms of configurability? From what there currently is, it seems like dprint is going for "make everything as reasonably customisable as possible, given it doesn't introduce whole new flows to the parser or printer", especially with overrides for specific AST node kinds since those are quite simple to implement.

Basically kind of a "build your own formatter" approach for people so trying to be configurable within what is reasonable to maintain. Something like adding a space somewhere is very easy to maintain so I'm open to all those.